src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/OppFinder/WhenHandlingGetOppFinderApprovedStandardDetailsRequest.cs

```diff
@@ -22,23 +24,31 @@ public class WhenHandlingGetOppFinderApprovedStandardDetailsRequest
 
         private GetOppFinderApprovedStandardDetailsHandler _sut;
 
+
+
         [SetUp]
-        public void Setup()
+        public async Task SetupAsync()
         {
             _mockOppFinderRepository = new Mock<IOppFinderRepository>();
             _mockMediator = new Mock<IMediator>();
             _mockLogger = new Mock<ILogger<GetOppFinderApprovedStandardDetailsHandler>>();
+            var validationRequest = new ValidationRequest()
+            {
+                Type = "email",
+                Value = "EMAIL",
+            };
 
-            _mockMediator.Setup(s => s.Send(It.Is<ValidationRequest>(x => x.Type == "email" && x.Value == "EMAIL"), It.IsAny<CancellationToken>()))
-                        .ReturnsAsync(true);
+            ValidationHandler validationHandler = new ValidationHandler(new ValidationService());
```

Comment:
See comment below first - but this would just be a complicted It.IsAny return false from a mock - which is strange as the original returned true - is it really being used?

Reply:
Original mock value for validationhandler was incorrect and not actually testing `EqaProviderName` logic, I updated mock of validationhandler with the values mentioned in my previous comment and added unit test to test `EqaProviderNam`e logic